templates/components/password.build.hbs

```diff
@@ -1,15 +1,19 @@
 <input
   type="password"
   class="form-control"
-  v-model.trim="value"
+  :class="{ disabled: autogenerate }"
+  :readonly="autogenerate"
+  autocomplete="off"
   v-on:input="updateValue()"
   :name="name"
   :id="name"
   :placeholder="placeholder"
   :required="required"
 />
 
-<div class="form-group row clearfix" v-if="confirm" :class="{ 'has-error': hasConfirmationError }">
+<p class="help-block" v-if="autogenerate">A password will be automatically generated</p>
```

Comment:
I thought the password field would have asterisks or dots in them. But it looks like you're not loading the field with the auto-generated passwords. I think you're doing this for security purposes so that the value can't be read? In this case, perhaps it's the easiest to just use `placeholder="A password will be automatically generated"` when `autogenerate` is enabled.

Reply:
That'll mean one less piece of UI on the screen, simpler design and less UI for theme customisation.

Reply:
@tonytlwu done 
![image](https://user-images.githubusercontent.com/574210/55076328-6bc60700-5095-11e9-894b-bda83c5f46aa.png)